deploy/application/php/tutorials/tutorial-wordpress.md

```diff
@@ -118,48 +183,12 @@ Redis should now work with your WordPress.
 
 {{< readfile "/content/partials/new-relic.md" >}}
 
-{{< readfile "/content/partials/env-injection.md" >}}
-
-{{< readfile "/content/partials/link-addon.md" >}}
-
-## Configure your database
-
-Make sure you have created a database add-on in the Clever Cloud console, and that it's linked to your application. When it's done, you will be able to access all of your add-on [environment variables](#setting-up-environment-variables-on-clever-cloud) from the application. You can use them as `DATABASE_URL=$MYSQL_ADDON_URI`.
-
-## Configure storage
-
-Create a FS Bucket add-on and link it to your application. Note its host (you can see it from the addon configuration panel, or in the environment variables exported by the addon). It looks like `bucket-01234567-0123-0123-0123-012345678987-fsbucket.services.clever-cloud.com`.
-
-Create a new [environment variable](#setting-up-environment-variables-on-clever-cloud) called `CC_FS_BUCKET` and set `/storage/app:<bucket-host>` as its value.
-
-At the root of your application, create a `clevercloud/buckets.json` file (create a `clevercloud` folder in which you create a `buckets.json` file).
-Add the following lines in this file. Do not forget to replace `bucketId` by the bucketId displayed in the [information]({{< ref "deploy/addon/fs-bucket.md" >}}) section of the FS Bucket add-on.
-    
-```javascript
-    [
-      {
-        "bucket" : "bucketId",
-        "folder" : "/wp-content/uploads"
-      }
-    ]
-```
-
-{{< readfile "/content/partials/deploy-git.md" >}}
-
-### Install a WordPress plugin with Git
-
-If you choose git deployment over FTP, the code of your plugins won't be tracked by git. This implies that you will not be able to install plugins from the administration panel and persist them between two deployments.
-
-To solve this problem, we recommend to install the plugin manually by copying the content of the plugin to the `/wp-content/plugins/` folder, add the new files to git and then deploy your application.
-
-The plugin will then be available in the **Extensions** section of your admin panel and you will be able to manage it as others WordPress plugins. 
-To uninstall the plugin, the procedure is the same as before except that you have to delete the folder corresponding to the plugin you want to delete. The extension will be automatically disabled, but we recommend you to delete it from you admin panel before removing the file, in order to clean your database and all files that the plugin could have created.
-
-
-{{< readfile "/content/partials/deploy-ftp.md" >}}
-
-{{< readfile "/content/partials/more-config.md" >}}
+## Deploy Wordpress with Bedrock
```

Comment:
"Bedrock" won't be explicit to our readers, rather write "Deploy WordPress the immutable way"